crates/sui-indexer/migrations/2023-10-06-204335_tx_indices/up.sql

```diff
@@ -3,47 +3,64 @@ CREATE TABLE tx_senders (
     tx_sequence_number          BIGINT       NOT NULL,
     -- SuiAddress in bytes.
     sender                      BYTEA        NOT NULL,
+    -- SystemTransaction/ProgrammableTransaction. See types.rs
+    transaction_kind            smallint     NOT NULL,
     PRIMARY KEY(sender, tx_sequence_number, cp_sequence_number)
 );
 CREATE INDEX tx_senders_tx_sequence_number_index ON tx_senders (tx_sequence_number, cp_sequence_number);
+CREATE INDEX tx_senders_transaction_kind_index ON tx_senders (sender, transaction_kind, tx_sequence_number, cp_sequence_number);
```

Comment:
`transaction_kind` is added to `tx_senders` and `tx_recipients` to facilitate queries filtered on address and transaction_kind.